### PR TITLE
fix: issues with cli reading from package.json

### PR DESCRIPTION
**Pull request by tylersayshi**
fix for https://github.com/dai-shi/waku/pull/978#issuecomment-2425461081

Also fixed an issue where the default folder name was not showing on conflict

### CODE REVIEW

File: packages/create-waku/src/index.ts [hunk 1]
```diff
@@ -161,7 +160,9 @@ Options:
 }
 
 async function notifyUpdate() {
-  const packageJson = JSON.parse(readFileSync('../package.json', 'utf8'));
+  const packageJson = JSON.parse(
+    readFileSync(path.resolve(import.meta.dirname, '../package.json'), 'utf8'),
```
Comment (dai-shi) [inline packages/create-waku/src/index.ts:164]: `import.meta.dirname` isn't available in Node 18.

https://nodejs.org/docs/latest-v18.x/api/esm.html#importmetaurl

Can you try this?
```js
import { readFileSync } from 'node:fs';
const buffer = readFileSync(new URL('./data.proto', import.meta.url));
```

**Review by dai-shi — COMMENTED** [review]
Thanks for the quick fix. See the comment.

File: packages/create-waku/src/index.ts (hunk 1 above)
Comment (tylersayshi) [reply packages/create-waku/src/index.ts]: yep, that works, thanks!

**Review by dai-shi — APPROVED** [review]
thx!

**Pull request merged**